mongodb_statedb/__init__.py

```diff
@@ -33,6 +33,20 @@ def get(self, key: str) -> Any:
         self._throw_error_if_key_doesnt_exist(key)
         return self._db.find_one({'_id': key})[self._VALUE_KEY]
 
+    def get_all_as_dict(self) -> dict:
```

Comment:
I don't think this is a good method to add. Part of the reason to have the get and set method is that the MongoDB server manages the data to make sure that it's always up to day. This approach creates its own object, which is a copy of the database and thus the two can diverge.

Reply:
I thought this would be a convenient method to have, but it turns out that I won't need to use this at all. I'll remove this one.